e2e/wdio/platform/tests/page-footer.e2e-spec.ts

```diff
@@ -0,0 +1,45 @@
+import { PageFooterPo } from '../pages/page-footer.po';
+import {
+    getElementArrayLength,
+    getText,
+    isElementClickable,
+    scrollIntoView,
+} from '../../driver/wdio';
+
+describe('PageFooter test suite:', function() {
+    const pagefooterPage = new PageFooterPo();
+    const {
+        pagefooterclicablelink, pagefootertext  } = pagefooterPage;
+
+    beforeAll(() => {
+        pagefooterPage.open();
+    }, 1);
+    
+        it('verify clickable links ', () => {
```

Comment:
this sounds like you are only checking a certain set of links. please rename to 'verify links are clickable'